src/drivers/amd/renoir/acp_dma.c

```diff
@@ -0,0 +1,621 @@
+// SPDX-License-Identifier: BSD-3-Clause
+//
+//Copyright(c) 2021 AMD. All rights reserved.
+//
+//Author:       Basavaraj Hiregoudar <basavaraj.hiregoudar@amd.com>
+//              Anup Kulkarni <anup.kulkarni@amd.com>
+//              Bala Kishore <balakishore.pati@amd.com>
+
+#include <sof/audio/component.h>
+#include<platform/chip_registers.h>
+#include<platform/fw_scratch_mem.h>
+#include<platform/chip_offset_byte.h>
+#include <sof/drivers/acp_dai_dma.h>
+#include <sof/drivers/timer.h>
+#include <sof/lib/alloc.h>
+#include <sof/lib/dma.h>
+#include <sof/lib/io.h>
+#include <sof/lib/notifier.h>
+#include <sof/lib/uuid.h>
+#include <sof/math/numbers.h>
+#include <sof/platform.h>
+#include <errno.h>
+#include <stddef.h>
+#include <stdint.h>
+
+
+#define MAX_STREAMS				8
```

Comment:
Can we prefix these macros with AMD_ as some are quite generic and may conflict with other generic macros..

Reply:
Addressed the Review Comment. Done